src/kbc_modules/cc_kbc/attester/tdx/mod.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::Attester;
+use anyhow::*;
+use serde::{Deserialize, Serialize};
+use std::path::Path;
+use tdx_attest_rs;
+
+const CCEL_PATH: &str = "/sys/firmware/acpi/tables/data/CCEL";
+
+pub fn detect_platform() -> bool {
+    Path::new("/dev/tdx-attest").exists()
```

Comment:
will this device auto-load in TDX VMs?
do you need to add it to some kata-gent config so it will be loaded by the agent?
or is it compiled as a built-in (`CONFIG...=y`) in the tdx kernel?

Reply:
This device will be automatically included and loaded in all guest kernel that support TDX. This is the core device for TDX attestation(If no `/dev/tdx-attest`, app can not perform TD Quote getting.), so it is appropriate to use it to detect the platform